presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -255,65 +255,65 @@ private TableHandle getTableHandle(ShowStats node, QualifiedName table)
                     .orElseThrow(() -> new SemanticException(MISSING_TABLE, node, "Table %s not found", table));
         }
 
-        private static List<String> findUniqueStatisticsNames(TableStatistics tableStatistics)
+        private static List<String> buildColumnsNames()
         {
-            TreeSet<String> statisticsKeys = new TreeSet<>();
-            statisticsKeys.addAll(tableStatistics.getTableStatistics().keySet());
-            for (ColumnStatistics columnStats : tableStatistics.getColumnStatistics().values()) {
-                statisticsKeys.addAll(columnStats.getStatistics().keySet());
-            }
-            return unmodifiableList(new ArrayList(statisticsKeys));
+            ImmutableList.Builder<String> columnNamesBuilder = ImmutableList.builder();
+            columnNamesBuilder.add("column_name");
+            columnNamesBuilder.add("data_size");
+            columnNamesBuilder.add("distinct_values_count");
+            columnNamesBuilder.add("nulls_count");
+            columnNamesBuilder.add("row_count");
+            return columnNamesBuilder.build();
```

Comment:
builder is better for so many key/values (i don't think there even is such an overload for `of`)